bin/dune

```diff
@@ -1,20 +1,20 @@
 (library
  (name      main)
- (modules   (main))
```

Comment:
This also applies to `(flags)` I guess - should we update the tests to remove the parentheses in these as well?

Reply:
Yes, I only fixed the build failures so far, but we could indeed update all the `flags` fields.

Reply:
Ah ok I didn't realise this was mandatory. IIUIC, an alternative would have been `(modules ("main"))`?

Reply:
yep